intro-HPC/ch_account.tex

```diff
@@ -259,6 +259,28 @@ \section{Getting ready to request an account}
   \strong{Without your key pair, you won't be able to apply for a personal VSC account.}
 \fi %macORlinux
 
+\iflinux
+
+\subsection{Using a SSH agent (optional)}
+\label{sec:using-ssh-agent-with-openssh}
+  The most recent GNU/Linux distributions include by default a SSH agent 
+  (``gnome-keyring-daemon'' in most cases) to keep and manage our SSH keys. 
+  For those distributions the user \strong{must} include the new key/s into 
+  the SSH manager keyring to be able to connect to the HPC cluster.
+  This could be done thanks to ``ssh-add'' command.
+
+\begin{prompt}
+Include the new key/s in our keyring:
+%\shellcmd{ssh-add}%
```

Comment:
@alvarosimon It's probably good to mention that `ssh-add` only works if the private key is in the default location at `~/.ssh/id_rsa`

if not, you need to do `ssh-add <path/to/private/key>`